english/marshmallow.md

```diff
@@ -606,4 +606,6 @@ Just then you recall the news story of what is causing the
 
 You escape into a [recording studio](more_cowbell/more_cowbell.md)
 
-And then you stumble upon something [completely different](http://www.atlasobscura.com/)
\ No newline at end of file
+And then you stumble upon something [completely different](http://www.atlasobscura.com/)
+
+A snarling ferret king who fixes you with a [steely glare](../ferrets/Ferret.md)
```

Comment:
It looks like this link is broken.  You can view the page on GitHub on your fork and click the link there to check if it is working correctly.